### ISSUE TITLE
Left Scroll Issue

### ISSUE THREAD

**Issue by deepjyoti30** [tarun-dugar/easy-scroll#8]
I am using the library to animate left and right scroll on the click of a button.

I have a horizontal scrolling div.

The right button works properly but whenever I use the left button, it scrolls all the way to the beginning of the list.

Here's the code I have for the right and the left button

```js
        scrollRight: function() {
            let view = document.getElementById(this.scrollId)
            this.scrollConfig["scrollableDomEle"] = view
            this.scrollConfig["scrollAmount"] = ( view.clientWidth / 20 ) * 5
            this.scrollConfig["direction"] = 'right';
            easyScroll(this.scrollConfig);
        },
        scrollLeft: function() {
            let view = document.getElementById(this.scrollId)
            this.scrollConfig["scrollableDomEle"] = view
            this.scrollConfig["scrollAmount"] = ( view.clientWidth / 20 ) * 5
            this.scrollConfig["direction"] = 'left';
            easyScroll(this.scrollConfig);
        },
```

I am using it in a Vue app.

Any help would be really appreciated. Also I checked your commits and saw that the latest release has 'left scroll fix' so I was just wondering if I am doing something wrong.

**Comment by DanielAlcaraz** [comment]
Hi @deepjyoti30, I'm having the same issue with the left scroll.

Any update on this?

**Comment by deepjyoti30** [comment]
@DanielAlcaraz Thank's about that. 

I've actually moved to using a complete carousel library instead of this, that makes life easier.

**Comment by tarun-dugar** [comment]
@deepjyoti30 @DanielAlcaraz fixed in 0.7.0
